pkgs/applications/networking/cluster/kns/default.nix

```diff
@@ -0,0 +1,33 @@
+{ stdenvNoCC, lib, fetchFromGitHub, bash }:
+
+stdenvNoCC.mkDerivation rec {
+  pname = "kns";
+  version = "unstable-2022-03-03";
+
+  src = fetchFromGitHub {
+    owner = "blendle";
+    repo = pname;
+    rev = version;
+    sha256 = "1y7m6ln3xricmrwf8638hqbmnbipfag1c1c04apfd4vcj7sqzm6b";
+  };
+
+  strictDeps = true;
+  buildInputs = [ bash ];
```

Comment:
Not really required. https://nixos.org/manual/nixpkgs/unstable/#sec-tools-of-stdenv

Reply:
sorry for the delay on this one, wedding + honeymoon + holidays got me all out of sync.

removed the `buildInputs`.

i tested with and without the `installPhase` and it looks like it _is_ required.

Reply:
https://github.com/NixOS/nixpkgs/pull/193995/commits/272b7cfca714f28b636f19f03ab438a1260c5604